tests/components/blue_current/test_init.py

```diff
@@ -61,10 +62,13 @@ async def test_config_exceptions(
     config_error: IntegrationError,
 ) -> None:
     """Test if the correct config error is raised when connecting to the api fails."""
-    with patch(
-        "homeassistant.components.blue_current.Client.validate_api_token",
-        side_effect=api_error,
-    ), pytest.raises(config_error):
+    with (
+        patch(
+            "homeassistant.components.blue_current.Client.validate_api_token",
+            side_effect=api_error,
+        ),
+        pytest.raises(config_error),
+    ):
         config_entry.add_to_hass(hass)
         await async_setup_entry(hass, config_entry)
```

Comment:
Side note: Please never call an integration function directly in tests unless it's an isolated *pure* helper function. We want to set up the integration and assert some core state like the config entry state.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations